_posts/2021-04-19-solving-problems-with-custom-extensions.adoc

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: 'Solving problems with custom Quarkus extensions'
+date: 2021-04-19
+tags: development-tips extensions
+synopsis: If fast startup and live reload weren't enough for you, maybe custom extensions will convince you that you can achieve great things with Quarkus.
+author: gsmet
+---
+
+From time to time, I see tweets or articles claiming they don't see the point of Quarkus because "who needs fast startup?", "I have plenty of memory" or "what is the point of live reload?".
+
+I could write an article to debunk these arguments and explain how the latter makes your development workflow much more efficient and how the former makes the latter possible, even if fast boots are not your thing.
+But for the sake of this blog post, let's admit these persons are absolutely right and they are not good reasons to use Quarkus.
+
+So now what? Back to `<insert your favorite framework here>`? Not so fast.
+
+Quarkus didn't achieve fast startup and low memory footprint by using dark magic or lazy loading tricks but by rethinking entirely the way Java applications are bootstrapped.
```

Comment:
I kinda feel the blog could just start here on line 18 - nothing technical wrong per say on the lines above but they give a bit of a negative vibe.